agot-bg-website/api/views.py

```diff
@@ -34,7 +34,34 @@ class RoomViewSet(CreateModelMixin, GenericViewSet):
 
 @api_view(['POST'])
 @csrf_exempt
-def notify(request, game_id):
```

Comment:
@Longwelwind Would you be so kind this part of the code together with the LiveWebSiteClient.ts ? I need no review for game server logic, just for this website part as I did it blind. Thank you in advance!

Of course, it would be better to define one `notify` endpoint accepting the `NotificationType` but this way was just easier to do it "blind".